.github/workflows/publish_packages.yml

```diff
@@ -0,0 +1,56 @@
+name: publish_packages.yml
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch:
```

Comment:
I think we only want to publish on tag release

Reply:
I think there might be cases where we want to trigger upload manually. I think we can keep it